mwzeval/metrics.py

```diff
@@ -293,11 +293,11 @@ def get_dst(input_data, reference_states, include_loocv_metrics=False, fuzzy_rat
     """
     DOMAINS = {"hotel", "train", "restaurant", "attraction", "taxi"}
 
-    def block_domains(input_states: dict, reference_states: dict, blocked_domains: set[str]) -> dict:
+    def block_domains(input_states: dict, reference_states: dict, included_domains: set[str]) -> dict:
         """Return new input and reference state dictionaries with the specified domains removed.
```

Comment:
Rephrase as follows:

Filter input and reference states to only include states from `included_domains`. This is useful for evaluating in the leave-one-out setup where the joint goal accuracy should be computed i) jointly with respect to the "left out" (aka unseen) domain and ii) jointly with respect to all other domains.

Turns whose references do not contain any slots from `included_domains` are dropped.